### ISSUE TITLE
Hide attached parts in palette

### ISSUE THREAD

**Issue by DavidGasku** [jmoenig/Snap--Build-Your-Own-Blocks#1999]
Hi, I would like the option to hide objects that are attached to other objects, or have the ability to manually hide them from the palette.
http://snap.berkeley.edu/snapsource/snap.html#present:Username=davidgasku&ProjectName=slider

For example in this project, I would like to have just one object (slider). The child parts are not needed to use the slider.
What do you think?

**Comment by bromagosa** [comment]
Hey, these SVG blocks are really cool! They should be in an official library!

I don't know about hiding sprites from the corral though. I'd prefer some sort of collapsible sprite collection, or a "3D" corral.

**Comment by DavidGasku** [comment]
Yes, something like that, an option in the parent sprite to collapse its children.
About the SVG, it would be an honor, but there are some problems to fix:
https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/issues/1743
